packages/eslint-plugin/lib/rules/prefer-button-component.js

```diff
@@ -0,0 +1,32 @@
+const jsxAstUtils = require('jsx-ast-utils');
+
+const { elementType, getProp, getLiteralPropValue } = jsxAstUtils;
+
+const MESSAGE =
+  'The <va-button> Web Component should be used to instead of the button HTML element.';
```

Comment:
Typo here and in the other one too: "The <va-button> Web Component should be used ~to~ instead of the button HTML element."